src/options/popup.js

```diff
@@ -27,15 +28,26 @@ export default {
   // each item defines the inner text for the element that it references
   content: {
     header : 'Cookies used on the website!',
-    message: 'This website uses cookies to ensure you get the best experience on our website.',
+    message: 'Oh no, you\'re thinking, yet another cookie pop-up. Well, sorry, it\'s the law. We measure how many people visit us and which parts are most interesteing, by storing cookies on your device. If you\'re cool with that, hit �Accept all Cookies�. For more info and to customise your settings, hit �Customise Settings�.',
     dismiss: 'Got it!',
-    allow  : 'Allow cookies',
+    allow  : 'Accept all Cookies',
     deny   : 'Decline',
     link   : 'Learn more',
     href   : 'https://www.cookiesandyou.com',
     close  : '&#x274c',
     target : '_blank',
-    policy : 'Cookie Policy'
+    policy: 'Cookie Policy',
+    customize: 'Customize settings',
+    customizeHeader: 'Review and manage your consent',
+    customizeMessage: 'Here is overview of cookies we use on this site. Select categories which you are OK with. You can also change your choices at any time, by hitting the �Your Consent Options� link on the site\'s footer.',
+    acceptSelected: 'Accept Selected',
+    categoryAnalytics: 'These cookies collect information to help us understand how our website is being used. They allow us to count visits and traffic sources so that we can measure and improve the content of our site.',
+    categoryEssential: 'These cookies are necessary so that you won\'t be seeing this pop-up next time you visit us.',
```

Comment:
Consider this:

> These cookies are necessary to make this site run properly and securely. For example, with this kind of cookies, we register your cookie preferences so that you won't be seeing this pop-up next time you visit our page and we can keep track which categories you have opted-in.
> To keep this site secure, we use Cloudflare (link) content delivery network and security solutions. The service may place a unique cookie to identify your browser and device to make sure no automated programs can impose security threats on our site.